convert_osm/src/gtfs.rs

```diff
@@ -0,0 +1,257 @@
+use std::collections::{BTreeMap, HashMap, HashSet};
+use std::fs::File;
+
+use anyhow::Result;
+use serde::Deserialize;
+
+use abstutil::MultiMap;
+use geom::{LonLat, PolyLine, Pt2D};
+use kml::{ExtraShape, ExtraShapes};
+use map_model::raw::{RawMap, RawTransitRoute, RawTransitStop};
+use map_model::PathConstraints;
+
+pub fn import(map: &mut RawMap) -> Result<()> {
+    // Collect metadata about routes
+    for rec in csv::Reader::from_reader(File::open(map.name.city.input_path("gtfs/routes.txt"))?)
```

Comment:
We just read the raw CSV files here. There are some crates that could maybe help with some of this work to join all the different files together -- https://docs.rs/gtfs-structures/0.29.0/gtfs_structures/struct.Gtfs.html or maybe https://docs.rs/transit_model/0.43.0/transit_model/model/struct.Collections.html -- but in the short amount of time I had to finish this PR, it didn't seem like either would help. Feel free to revisit that decision.